shared/charts/couchdb/templates/network_policy.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-{{ template "couchdb.fullname" . }}-to-{{ template "couchdb.fullname" . }}
+  labels:
+    app: {{ template "couchdb.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  podSelector:
+    matchLabels:
+      app: {{ template "couchdb.name" . }}
+      release: {{ .Release.Name }}
+  policyTypes:
+  - Ingress
+  ingress:
+  - from:
+    - podSelector:
+        matchLabels:
+          app: {{ template "couchdb.name" . }}
+          release: {{ .Release.Name }}
```

Comment:
This policy can be made more strict by limiting ingress to only CouchDB ports: 5984, 4369 and 9100.

Reply:
I wasn't sure about the ports, but makes sense to be consistent with the rest of the policies. 👍